api/src/main/java/org/pmiops/workbench/rdr/RdrExportEnums.java

```diff
@@ -70,8 +72,8 @@
   private static final BiMap<Disability, org.pmiops.workbench.rdr.model.Disability>
       CLIENT_TO_RDR_DISABILITY =
           ImmutableBiMap.<Disability, org.pmiops.workbench.rdr.model.Disability>builder()
-              .put(Disability.TRUE, org.pmiops.workbench.rdr.model.Disability.TRUE)
-              .put(Disability.FALSE, org.pmiops.workbench.rdr.model.Disability.FALSE)
+              .put(Disability.TRUE, org.pmiops.workbench.rdr.model.Disability.YES)
```

Comment:
This confuses me. Either we have a boolean, in which case we don't need an enum, or we have an enum that describes  the situation like `Disability.NOT_DISABLED`, `Disability.SOME`, etc. But if you want to start an enum, it's hard to imagine what you'd add to yes and no.

Reply:
We basically have three options for DISABILITY FOR RDR its YES, NO ,PREFER_NOT_TO_ANSWER